Documentation/faq.md

```diff
@@ -1,5 +1,11 @@
 ## Frequently Asked Questions (FAQ)
 
+### etcd, general
+
+#### Does the clients need to send requests to the leader?
```

Comment:
does our client retry a different endpoint if the initial endpoint is not reachable?